truss/config.go

```diff
@@ -18,4 +18,7 @@ type Config struct {
 	DefPaths []string
 	// The files of a previously generated service, may be nil
 	PrevGen map[string]io.Reader
+
+	// The flag that specifies if a service should be built.
+	NoService bool
```

Comment:
If it would be preferable for `parseInput` to return an additional value (`NoService`) rather than extending the `Config` struct I can rework that.